setup.py

```diff
@@ -4,8 +4,8 @@
     name='django-autocompleter',
     version="0.6.0",
     description='A redis-backed autocompletor for Django projects',
-    author='Ara Anjargolian',
```

Comment:
Woah... Calm down cowboy ... you can do this tho https://stackoverflow.com/questions/9999829/how-to-specify-multiple-authors-emails-in-setup-py